proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/structure/MetsElementVisitor.java

```diff
@@ -199,6 +199,29 @@ protected void initHeader(IMetsElement metsElement) throws MetsExportException {
     }
 
     /**
+     * Returns the name of title if element is issue
+     */
+    private String getTitle(IMetsElement metsElement) throws MetsExportException {
+        if (isIssue(metsElement)) {
+            Node partNode = MetsUtils.xPathEvaluateNode(metsElement.getModsStream(), "//*[local-name()='mods']/*[local-name()='titleInfo']/*[local-name()='title']");
+            if (partNode == null){
+                throw new MetsExportException("Error - missing title. Please insert title.");
+            }
+            return partNode.getTextContent() + " ";
+        }
+        return "";
+    }
+
+    /**
+     * Returns true if element is issue, else return false
+     */
+    public boolean isIssue(IMetsElement metsElement) throws MetsExportException {
```

Comment:
Ano, jde jen o NdkPeridicalIssue tedy číslo periodika. 
Pokud mám založený jen titul periodika nebo titul + ročník ale žádné číslo ani žádnou přílohu a chci exportovat tuto strukturu (titul, titul + ročník) export vždy dopadne tak, že se pouze založí složka a do ní se vygeneruje soubor s logem. Žádné jiné soubory v této složce nejsou. Pokud chci  i jiné soubory musím založit číslo periodika nebo přílohu. Potom se již vygeneruje do složky MD5, Info a METS soubor. A do METS souboru se právě promítne tato změna